### DISCUSSION TITLE
I have a question

### DISCUSSION THREAD

**Discussion by Thomas282-coder** [ScratchAddons/ScratchAddons#1060] [Help] [answered]
I'm trying to translate scratch addons, but what is **onion skinning**

**Comment by mxmou** [comment, marked as answer]
https://en.wikipedia.org/wiki/Onion_skinning

**Reply by WorldLanguages** [reply]
Yeah, @McGithub312 feel free to translate the term as you prefer, no need to translate literally. You can also add `(onion skinning)` without parenthesis after the translation, even if Transifex warns you for doing so

**Reply by Thomas282-coder** [reply]
Ok.
